### PR TITLE
[FIG] feat: build specs on master merge

### PR DESCRIPTION
**Pull request by cstrnt**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behaviour? (You can also link to an open issue here)**
We rely on users actually running `npm run build` and also checking the `.js` files in. This has quite some potential for errors

**What is the new behaviour (if this is a feature change)?**
On a new push to master (merged PR / push to master) all specs are built and a new release is created. The artifact can then be downloaded using the following command:
This also allows easier version for the autocompletion specs (e.g. knowing what version of specs they have; makes debugging easier)

**Additional info:**

```bash
# Get the latest release and download the zipped specs 
curl -Lo specs.zip https://github.com/withfig/autocomplete/releases/latest/download/specs.zip \
# unzip them in the current directory
&& unzip ./specs.zip \
# remove the `.zip`file
&& rm -f specs.zip
```

### CODE REVIEW

**Comment by falky97** [comment]
Before this is merged in we need to update `fig update` and the installation script so we pull down the specs correctly...

**Comment by falky97** [comment]
Potentially update this merge to update the gitignore change and create a separate PR for this

**Review by mattschrage — APPROVED** [review]
Other than the small change, I think it looks good.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      - name: bundle files
+        run: zip -r specs.zip specs/
+        # Bump the version and create a git tag with the new version
+      - name: Bump version and push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v5.4
+        with:
+          tag_prefix: spec-build-no#
+          # Bump versions in full numbers (1.0.0 -> 2.0.0)
+          default_bump: major
```
[the remark this answers is not in the document]
Comment (cstrnt) [reply .github/workflows/build.yml:33]: I think Brendan said in our call that he just wanted full number increases. At least I wrote it down during our meeting. @falky97 are you fine with smaller increments as well?

**Pull request merged**